crates/re_selection_panel/src/defaults_ui.rs

```diff
@@ -0,0 +1,241 @@
+use std::collections::{BTreeMap, BTreeSet};
+
```

Comment:
quite a bit of unfortunate copy & paste in here from component override ui, but probably better than the work intensive alternative with dubious future outlook :+1:

Reply:
That was my thought -- I think they both get thrown away and replaced with something universal.